test/cli_test.rb

```diff
@@ -291,6 +291,31 @@
     end
   end
 
+  describe 'secret' do
+    describe 'when `secret` command' do
```

Comment:
The second describe line is a little redundant, since there is only the command and no other case, right?